dali/pipeline/executor/executor.h

```diff
@@ -118,6 +131,19 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
   DISABLE_COPY_MOVE_ASSIGN(Executor);
 
  protected:
+  using mem_stats_map = std::unordered_map<std::string, std::vector<std::pair<size_t, size_t>>>;
```

Comment:
Please use some meaningful struct instead of pair of numbers.

Reply:
Done